cmake/faceliftMacros.cmake

```diff
@@ -584,6 +584,9 @@ function(facelift_add_library TARGET_NAME)
                 set(LIBRARY_TYPE INTERFACE)
             endif()
             add_library(${TARGET_NAME} ${LIBRARY_TYPE} ${ALL_SOURCES})
```

Comment:
before calling add_library, simply do:

if(DO_NOT_RELINK_SHARED)
    set(CMAKE_LINK_DEPENDS_NO_SHARED ON)
endif()

That should be all you need.

Reply:
done